modules/nf-commons/src/main/nextflow/util/CmdLineOptionMap.groovy

```diff
@@ -0,0 +1,63 @@
+package nextflow.util
+
+/**
+ * Holder for parsed command line options.
+ *
+ * @author Manuele Simi <manuele.simi@gmail.com>
+ */
+class CmdLineOptionMap {
```

Comment:
Whenever possible new classes should be annotated with `@CompileStatic`

Reply:
It's missing `CmdLineOptionMapTest` or I'm wrong?

Reply:
Indeed, it was missing. Just added.